pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
+	if err != nil {
+		return restoreWaitUnknown
+	} else if strings.Contains(string(bytes), "[-w secs]") {
```

Comment:
I meant we should just backport the --version patch to RHEL too, and then that's why I said "if it prints *anything*, you know it's good"; just assume that any version of iptables-restore that supports --version also supports -w, regardless of the actual version number.

(If that sounds too crazy to you/other reviewers then I guess check for 1.6.2 in kubernetes and we can patch it somehow in OpenShift.)